Runtime/Scripts/LocationService.cs

```diff
@@ -40,18 +39,15 @@ public static IEnumerator InitalizeLocationService(int maxWait = 20, bool contin
             // First, check if user has location service enabled
             if (!Input.location.isEnabledByUser)
             {
-
 #if UNITY_IOS
                 //if isEnabledByUser is false and you start location updates anyway,
                 //the CoreLocation framework prompts the user with a confirmation panel
                 //asking whether location services should be reenabled.
                 //The user can enable or disable location services altogether from the Settings application
                 //by toggling the switch in Settings>General>LocationServices.
                 //https://docs.unity3d.com/ScriptReference/LocationService-isEnabledByUser.html
-#elif UNITY_EDITOR
-                Debug.LogWarning("MobiledgeX: Location Service disabled in UnityEditor");
 #else
-                throw new Exception("MobiledgeX: Location Service disabled by user."); // 
+                locationPermissionRejected = true;
```

Comment:
I think each time you set locationPermissionRejected to true in this function and in LocationServiceFlow(), we should Input.location.Stop(); and then return, right?